pages/vendors.tsx

```diff
@@ -35,18 +36,23 @@ function Vendors(props: VendorProps):JSX.Element {
     return (
         <>
             <Nav />
-            <Heading
-                mainTitle="Vendors"
-                subTitle="All available vendors."
-            />
-            {
-                data.vendors
-                    .map((vendor: Vendor) =>
-                        <h1>{vendor.name}</h1>
-                    )
-            }
-
-            <Footer />
+            <div className="container">
+                <Heading
+                    mainTitle="Vendors"
+                    subTitle="All available vendors."
+                />
+
+                <div className="images-container">
+                    {
+                        data.vendors
+                            .map((vendor: Vendor) =>
+                                <VendorCard {...{vendor}} key={vendor._id} />
```

Comment:
or use `vendor={vendor}` or destruct in component.
```suggestion
                                <VendorCard {...vendor} key={vendor._id} />
```